js_modules/dagster-ui/packages/ui-core/src/workspace/WorkspaceContext.tsx

```diff
@@ -447,6 +447,11 @@ export const useRepository = (repoAddress: RepoAddress | null) => {
   return findRepositoryAmongOptions(options, repoAddress) || null;
 };
 
+export const useJob = (repoAddress: RepoAddress | null, jobName: string | null) => {
```

Comment:
instead of `| null` you could do 

```
(repoAddress?: RepoAddress, jobName?: string) => {
```

then you can just do to do 
`useJob(repoAddress, explorerPath.pipelineName)?.pipelineSnapshotId;`

Reply:
I was just emulating whats above on `useRepository` wasn't sure why it was that way

Reply:
oh, yeah not sure. I guess it's a preference of `null` over `undefined` ?